### ISSUE TITLE
TupleElementNameAttribute missing reference when building on Linux

### ISSUE THREAD

**Issue by lauromoura** [dotnet/docfx#5160]
**Operation System**: Ubuntu x86-64 (18.04 and 19.04)

Mono version: 6.4.0.198 from Mono repositories
Dotnet version: 3.0.11 (although it seems to be using mono's Msbuild)

**DocFX Version Used**: 2.45 tarball

**Steps to Reproduce**:

1. Run build script (after installing powershell, npm, etc)

**Expected Behavior**:

docfx built

**Actual Behavior**:

Build fails with

```
"/home/lauro/dev/sources/docfx/docfx-2.45.1/src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj" (default target) (25) ->              
(CoreCompile target) ->                                                                                                                                                              
  ExtractMetadata/CompilationUtility.cs(80,35): error CS8137: Cannot define a class or member that utilizes tuples because the compiler required type 'System.Runtime.CompilerServices.TupleElementNamesAttribute' cannot be found. Are you missing a reference? [/home/lauro/dev/sources/docfx/docfx-2.45.1/src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj]
  ExtractMetadata/CompilationUtility.cs(80,35): error CS8356: Predefined type 'System.ValueTuple`2' is declared in multiple referenced assemblies: 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' and 'System.ValueTuple, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' [/home/lauro/dev/sources/docfx/docfx-2.45.1/src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj]   
```

**Comment by dlech** [comment]
Had the same problem on Mac. Got it to build with this change:

```diff
diff --git a/src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj b/src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj
index 92a76a1aa..10aa499ed 100644
--- a/src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj
+++ b/src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj
@@ -22,5 +22,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Locator" Version="1.2.2" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
 </Project>
```

**Comment by superyyrrzz** [comment]
Sorry for the later response... @dlech Could you try adding it in `docfx.csproj`? DocFX experienced similar issues and already added some references there, like [`SQLitePCLRaw.core`](https://github.com/dotnet/docfx/blob/2c48800d81317470b8f70d340f3049999aac6011/src/docfx/docfx.csproj#L45).

**Comment by yufeih** [comment]
Fixed in the [v2.60.0](https://github.com/dotnet/docfx/discussions/8334) release that runs on .NET Core with support for MacOS and linux without mono.
